statistics/histogram.go

```diff
@@ -392,10 +392,29 @@ func (hg *Histogram) getIncreaseFactor(totalCount int64) float64 {
 	return float64(totalCount) / float64(columnCount)
 }
 
+func validRange(ran *ranger.NewRange) bool {
+	var low, high []byte
+	if ran.LowVal[0].Kind() == types.KindBytes {
+		low, high = ran.LowVal[0].GetBytes(), ran.HighVal[0].GetBytes()
+	} else {
+		low, high = codec.EncodeInt(nil, ran.LowVal[0].GetInt64()), codec.EncodeInt(nil, ran.HighVal[0].GetInt64())
```

Comment:
Can we just compare the int value instead of encode it?

Reply:
Then we need to consider carefully the four combinations of low exclude and high exclude?